src/calculateRentalCost.js

```diff
@@ -3,8 +3,25 @@
  *
  * @return {number}
  */
+
+const PRICE_PER_DAY = 40;
+const LONG_TERM = 7;
+const SHORT_TERM = 3;
+const BIG_DISCOUNT = 50;
+const SMALL_DISCOUNT = 20;
+
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE = days * PRICE_PER_DAY;
```

Comment:
```suggestion
  const currentPrice = days * PRICE_PER_DAY;
```
Variable names should be in camelCase